server/routes/index.js

```diff
@@ -164,6 +165,16 @@ module.exports = function Index ({ authenticationMiddleware }) {
     const templateValues = await getCaseAndTemplateValues(req)
     templateValues.title = 'Risk register'
 
+    const riskResponse = await getRiskDetails(req.params.caseNo)
```

Comment:
We need to pass the CRN to the community endpoint to retrieve risk data.

Reply:
Yep, I'll sort